main/SQLHelper.cpp

```diff
@@ -5388,7 +5411,7 @@ void CSQLHelper::AddCalendarUpdateUV()
 			//insert into calendar table
 			result=safe_query(
 				"INSERT INTO UV_Calendar (DeviceRowID, Level, Date) "
-				"VALUES ('%llu', '%g', '%q')",
+				"VALUES ('%llu', '%.g', '%q')",
```

Comment:
I still see a lot of non-needed changes, and for example this one (procent point 'g') is not what it should be
Maybe best to revert this whole file, and only make that change needed for your patch